app/containers/UnlockEtherContainer.tsx

```diff
@@ -117,40 +157,57 @@ class UnlockEtherContainer extends React.Component<IUnlockEtherContainerProps> {
       <div className={styles.section}>
         {willBePenalized
           ? <Alert bsStyle="danger">
-              Withdrawing funds now will result in penalty of 10% of your funds!
+              <Checkbox onChange={this.changeAgreeToPayPenalty} checked={this.state.agreeToPayFee}>
+                I confirm that by following unlock procedure I agree to return{" "}
+                <MoneyComponent
+                  value={this.props.neumarkNeededToUnlockEth}
+                  tokenType={TokenType.NEU}
+                />{" "}
+                and to pay <MoneyComponent value={penalty} tokenType={TokenType.ETHER} /> unlock fee
```

Comment:
`and to pay <MoneyComponent value={penalty} tokenType={TokenType.ETHER} /> unlock fee` should be displayed only when `willBePenalized === true`